meta.yaml

```
@@ -44,6 +44,7 @@ requirements:
     - transformers >=4.11,<5
     - datasets >=1.14,<2
     - pycocotools >=2.0.4,<3
+    - boto3>=1.21.45
```

Nit: Sorry, this will re-trigger the 30 min conda build, but max versions are needed (thanks HF for the lesson learned!)

```suggestion
    - boto3 >=1.21.45,<2
```

---

Ah yep! No worries, I remember the hassle this caused during release a few months back. Will fix.